src/zap/zapimage.cpp

```diff
@@ -1547,7 +1547,7 @@ void ZapImage::OutputTables()
 
 #if defined(FEATURE_PAL)
     // PAL library requires native image sections to align to page bounaries.
-    SetFileAlignment(0x1000);
+    SetFileAlignment(0x10000);
```

Comment:
I believe that this will make native images a lot bigger. I am not sure whether this would be acceptable.

How are the native binary formats like ELF dealing with the 64k pages? Do they have the sections aligned at 64k boundaries as well?